### PR TITLE
Basic Spider Module

### PR DESCRIPTION
**Pull request by Razzy52**
Spider module with two modes ( Vanilla and Vulcan)

### CODE REVIEW

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/spider/modes/SpiderVulcan286.kt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import net.ccbluex.liquidbounce.event.repeatable
+import net.ccbluex.liquidbounce.features.module.modules.movement.spider.ModuleSpider
+
+
+    /*
+    * Vulcan mode for the Spider module.
+    * Made for Vulcan286
+    * Tested on Eu.loyisa.cn and Anticheat-test.com
+    * It may still flag sometimes, particularly when going more then 15-30 blocks up.
+     */
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/spider/modes/SpiderVulcan286.kt]: weird spacing

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/spider/ModuleSpider.kt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with LiquidBounce. If not, see <https://www.gnu.org/licenses/>.
+ */
+package net.ccbluex.liquidbounce.features.module.modules.movement.spider
+
+import net.ccbluex.liquidbounce.features.module.Category
+import net.ccbluex.liquidbounce.features.module.Module
+import net.ccbluex.liquidbounce.features.module.modules.movement.spider.modes.*
+
+object ModuleSpider : Module("Spider", Category.MOVEMENT) {
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/spider/ModuleSpider.kt]: add alias WallClimb or rename module to WallClimb and add alias Spider. In Legacy it was named WallClimb

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/spider/modes/SpiderVulcan286.kt [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            tickCounter = 0
+            return@repeatable
+        }
+
+        if (tickCounter >= 3)
+            tickCounter = 0
+
+        tickCounter++
+
+        if (player.horizontalCollision && tickCounter == 2 % 3) {
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/spider/modes/SpiderVulcan286.kt]: I feel like we could have used waitTicks(...) a lot here to simplify the code. Also please put brackets, it just looks better.

**Comment by superblaubeere27** [comment]
Btw, you can check if your commits will build with `gradle build` or `gradle detekt`

**Pull request merged**